### ISSUE TITLE
Test Needed 

### ISSUE THREAD

**Issue by lemyskaman** [lemyskaman/lorem-ipsum#1]
It's necessary to create a test for the LorempIpsum class

**Comment by lemyskaman** [comment]
> It's necessary to create a test for the LorempIpsum class

@mnpenner do you think you can create one?, honestly, I don't know-how.

**Comment by mnpenner** [comment]
No, sorry. Take a look at PhpUnit. There should be some examples.

**Comment by lemyskaman** [comment]
thanks :)
